.gitlab/corona-templates.yml

```diff
@@ -19,7 +19,8 @@
       when: never
     - if: '$CI_JOB_NAME =~ /release_resources/'
       when: always
-    - when: on_success
+    - if: '$LC_ZONE == "CZ"' #run only if ...
+      when: always
```

Comment:
here is my suggestion:
```
  rules:
    - if: '$ON_CORONA == "OFF"'
      when: never   # no jobs on corona if ON_CORONA == OFF
    - if: '$LC_ZONE ~= "CZ"' #run only on CZ ...
      when: never   # no jobs on corona if not on CZ
    - if: '$CI_JOB_NAME =~ /release_resources/'
      when: always   # release_resources jobs should run whatever the previous stage status
    - when: on_success   # regular jobs need one true if clause to run  
```

Reply:
I think I have incorporated this in to my yaml files. Hopefully I will get these pipeline rules figured out soon! Thanks for the example!

Reply:
There is a typo in my example I found in your code (my bad...): the proper syntax for "not equal" is `=~`.

Reply:
Oh I see haha I should have caught that!